sdk-extension/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2017 Hewlett-Packard Enterprise Development Company, L.P.
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>sdk-root</artifactId>
+        <groupId>com.hpe.adm.nga.sdk</groupId>
+        <version>12.55.5-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.hpe.adm.nga.sdk.extension</groupId>
+    <artifactId>sdk-extension-root</artifactId>
+    <packaging>pom</packaging>
+
+    <name>HPE ALM Octane REST API SDK Extension</name>
+    <description>
+        Extension of ALM Octane Java REST SDK (https://github.com/HPSoftware/ALMOctaneJavaRESTSDK),
+        can be used for more advanced use cases.
+    </description>
+    <url>https://github.com/HPSoftware/sdk-extension</url>
+
+    <modules>
+        <module>sdk-src</module>
```

Comment:
I think the extensions should have different module names. Otherwise there could be confusion between the main sdk-src and the extension sdk-src

Reply:
Crap, I missed that...

Reply:
renamed the folder and submodule